sonar-db/src/main/java/org/sonar/db/measure/custom/CustomMeasureDao.java

```diff
@@ -42,14 +40,13 @@ public void delete(DbSession session, long id) {
     mapper(session).delete(id);
   }
 
-  public void deleteByMetricIds(final DbSession session, final List<Integer> metricIds) {
-    DatabaseUtils.executeLargeInputsWithoutOutput(metricIds, new Function<List<Integer>, Void>() {
-      @Override
-      public Void apply(@Nonnull List<Integer> input) {
+  public void deleteByMetricIds(DbSession session, List<Integer> metricIds) {
+    DatabaseUtils.executeLargeInputsWithoutOutput(
+      metricIds,
+      input -> {
         mapper(session).deleteByMetricIds(metricIds);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Make this line start at column 7. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AIndentationCheck)